data/Satyxm.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Satyam Singh",
+  "type": "personal",
+  "bio": "Learning New Tech Stacks and love contributing to the Open Source and learn by contributing. Evolving!",
+  "avatar": "https://github.com/Satyxm.png",
```

Comment:
Hi @Satyxm, thank you for contributing :tada: 
You can delete this line, it is not required anymore :+1: 
```Suggestion
```